lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -147,6 +148,17 @@
     }
 }
 
-.resource-edit > div {
+.resource-edit div {
```

Comment:
These can be merged with the rules in line 140 as well

Reply:
Please see the comment above. From the options I have tested either changes need to be made in both the handlebars file with the corresponding selector in the scss file or a css selector in the scss file alone. This sometimes results in nested selectors but we currently have Ember linting at a nesting depth of 3. 

Preference has been given by Brian Geiger to use the former. The various nested components in the handlebars file made this the most concise option I found. If there is another suggestion that will work without breaking the other elements’ layout or several changes that achieve the same or better result, please let me know.

Reply:
In that case, we can always take out the rules for `.resource-edit` from line 141-153 out of the `.container` rule, and move it to the top level with these rules. That shouldn't cause any functional changes, while keeping all the rules that apply to `.resource-edit` in a single place. Just my preference, so I'll leave that decision up to you

Reply:
Ok, these have been condensed and unnecessary code removed. Let me know if this is what you had in mind.